app/routes/_gcn.synonyms/synonyms.server.ts

```diff
@@ -40,3 +40,52 @@ export async function createSynonyms(...synonymousEventIds: string[]) {
 
   return uuid
 }
+
+/*
+ * If an eventId already has a synonym and is passed in, it will unlink the
+ * eventId from the old synonym and the only remaining link will be to the
+ * new synonym.
+ *
+ * BatchWriteItem has a limit of 25 items, so the user may not add and/or remove
+ *  more than 25 synonyms at a time.
+ */
+export async function putSynonyms({
+  uuid,
+  additions,
+  subtractions,
+}: {
+  uuid: string
```

Comment:
I had envisioned the uuid as an implementation detail that is not exposed in the API.

Reply:
No, it's not a problem. My original design in #745 did not use it. I was just remarking on a difference in design; that is all.